pkg/webhooks/user/user.go

```diff
@@ -149,19 +230,59 @@ func (s *UserWebhook) HandleRequest(w http.ResponseWriter, r *http.Request) {
 		responsehelper.SendResponse(w, resp)
 		return
 	}
+	// load users in groups to know who is allowed an account
+	// TODO (lisa): Cache this and/or periodically refresh the data
+	if err = s.loadUsers(); err != nil {
```

Comment:
This is possibly expensive.  Recommend move to where we really need it inside `isProtectedRedHatAssociate` to minimize calls out of the webhook to load data.